Readability.js

```diff
@@ -867,6 +878,41 @@ Readability.prototype = {
           lastScore = parentOfTopCandidate.readability.contentScore;
           parentOfTopCandidate = parentOfTopCandidate.parentNode;
         }
```

Comment:
Can we just replace this method of determining a better candidate by using an ancestor node with the method you're adding? If not, why not?